app/assets/stylesheets/modules/home.css

```diff
@@ -49,6 +49,7 @@
   margin-left: auto;
   position: relative;
   width: 90%;
+  margin-top: 30px;
```

Comment:
What do you think about moving this one a couple of lines to the top (Underneath line 49), that way the `margin`s are grouped together.